### ISSUE TITLE
Can't setup a VPC peering with VPC in other account

### ISSUE THREAD

**Issue by hashibot** [terraform-providers/terraform-provider-aws#582]
_This issue was originally opened by @FransUrbo as hashicorp/terraform#12448. It was migrated here as part of the [provider split](https://www.hashicorp.com/blog/upcoming-provider-changes-in-terraform-0-10/). The original body of the issue is below._

<hr>

If I use `auto_accept = false`, then the peering connection will be created (but not setup correctly) and I go and accept the peering in the remote account.

Next run TF runs, it will fail to modify it:

```
aws_vpc_peering_connection.test-access: Modifying...
  requester.1102046665.allow_classic_link_to_remote_vpc: "" => "false"
  requester.1102046665.allow_remote_vpc_dns_resolution:  "" => "true"
  requester.1102046665.allow_vpc_to_remote_classic_link: "" => "false"
  requester.3244290774.allow_classic_link_to_remote_vpc: "false" => "false"
  requester.3244290774.allow_remote_vpc_dns_resolution:  "false" => "false"
  requester.3244290774.allow_vpc_to_remote_classic_link: "false" => "false"
[...]
* aws_vpc_peering_connection.test-access: Error modifying VPC Peering Connection options: OperationNotPermitted: User LOCAL_ACCOUNT_ID does not have permission to modify the accepter side peering options
        status code: 400, request id: 348f0ae0-21ca-4caf-ba30-fdeb07e8a109
```

### Terraform Version
0.8.5

### Affected Resource(s)
- aws_vpc_peering_connection

### Terraform Configuration Files
```hcl
resource "aws_vpc_peering_connection" "test-access" {
  vpc_id                            = "${aws_vpc.test.id}"

  peer_vpc_id                       = "${data.terraform_remote_state.core.access_vpc_id}"
  peer_owner_id                     = "${data.terraform_remote_state.core.account_id}"

  auto_accept                       = "false"

  accepter {
    allow_remote_vpc_dns_resolution = true
  }

  requester {
    allow_remote_vpc_dns_resolution = true
  }

  tags {
    Name                            = "test-access"
    environment                     = "test"
    location                        = "test"
    service                         = "network"
  }
}
```

### Expected Behavior
TF should have created the peering connections with all the correct variables, attributes and other settings (but left it non-accepted), so when the user (me) goes into the remote account and accepts them, they're "all ready to go".

### Actual Behavior
The peering connection was only partially created, requiring a modification of it after it had been accepted manually. Which fails because of lack of access rights.

**Comment by ildanish** [comment]
I'm having exactly the same problem, my use case is that I need to create VPC peering connections from the account I own to VPCs in accounts I don't manage, so I can't use the aws_vpc_peering_connection_accepter resource as I don't have the credentials for the accepter side.

I'm using terraform 0.10.7

Is there any plan to fix this?

**Comment by ewbankkit** [comment]
This has been fixed with the [`aws_vpc_peering_connection_options`](https://www.terraform.io/docs/providers/aws/r/vpc_peering_options.html) resource added via https://github.com/terraform-providers/terraform-provider-aws/pull/3909 and released in [version 1.17.0 of the AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#1170-may-02-2018).

**Comment by bflad** [comment]
Thanks @ewbankkit -- For anyone finding this please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.
